src/__tests__/index.js

```diff
@@ -701,10 +704,10 @@ test('appends to root plugins array', async () => {
     }),
   )
 
-  expect(optionRootFoo).toHaveBeenCalledTimes(10)
+  expect(optionRootFoo).toHaveBeenCalledTimes(13)
   expect(optionFoo).toHaveBeenCalledTimes(2)
   expect(optionBar).toHaveBeenCalledTimes(1)
-  expect(programVisitor).toHaveBeenCalledTimes(11)
+  expect(programVisitor).toHaveBeenCalledTimes(14)
```

Comment:
Do you know why these functions would get called more due to these changes?

Reply:
Because of adding more fixtures

Reply:
Oh, sorry, I was confused. Thanks!